az-digital/az_quickstart
# Closes #2536, Closes #2876: Add missing Intelligence Bank setting and set media embed triggered modal width to 75% of browser window.

## Description

I've updated the code for intelligencebank to fix the issue in #2876 .



## Related issues




- Closes #2536
- Closes #2876

## How to test


Check if DAM login works on site that had the `az_digital_asset_library` module installed previously.
Edit an existing text area
Click insert media in the ckeditor toolbar
[Click "Add Arizona Digital Asset Library Media"](https://69220f2d-6259-4384-9ec9-c8b08a787cae.probo.build/user/login)
## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
